PHP/Language.md

```diff
@@ -97,17 +111,23 @@ $_GET, $_POST, $_REQUEST
 
```

Comment:
I would ask about difference $_GET, $_POST, $_REQUEST

Reply:
Yeah, It's basics. Actually, did not include them in the first edition, as nowadays all people used to work with Request objects. But will add now.